### ISSUE TITLE
With Block default member object compiler crash

### ISSUE THREAD

**Issue by Kr00l** [WaynePhillipsEA/twinbasic#382]
**Describe the bug**
below code block works in VBx. However, in tB only the `MsgBox Var(3)` works and the compiler crashes on `MsgBox .This(3)`

**To Reproduce**
```
Class MyClass
Public Property Get DefaultMember(ByVal Index As Long) As Object
Attribute DefaultMember.VB_UserMemId = 0
    Debug.Print Index
    Set DefaultMember = New StdFont
End Property
End Class

Module MainModule

Private Type TMyStruct
This As Object
End Type
Public Sub Main()
    
    Dim MyStruct As TMyStruct, Var As Object
    
    With MyStruct
    Set .This = New MyClass
    
    MsgBox .This(3) ' Should show 'Arial', but compiler crashes
    
    Set Var = .This
    MsgBox Var(3) ' this works and show 'Arial'
    
    End With
    
End Sub

End Module
```

**Screenshots**
n/a

**Desktop (please complete the following information):**
 - OS: [e.g. Windows 10]
 - twinBASIC compiler version [e.g. 0.10.4756]

**Additional context**
Having no "Index" param on the default member works. See below amended code block
So there must be an issue with the With Block and the `Index As Long` parameter on the default member.
```
Class MyClass
Public Property Get DefaultMember() As Object
Attribute DefaultMember.VB_UserMemId = 0
    Set DefaultMember = New StdFont
End Property
End Class

Module MainModule

Private Type TMyStruct
This As Object
End Type
Public Sub Main()
    
    Dim MyStruct As TMyStruct, Var As Object
    
    With MyStruct
    Set .This = New MyClass
    
    MsgBox .This ' this works and show 'Arial'
    
    Set Var = .This
    MsgBox Var ' this works and show 'Arial'
    
    End With
    
End Sub

End Module
```

**Comment by WaynePhillipsEA** [comment]
Thanks for reporting, this will be fixed today.   Changing `This As Object` to `This As Variant` is a temporary workaround.

**Comment by WaynePhillipsEA** [comment]
Fixed in v0.10.4904, thanks!
